cpp/src/barretenberg/honk/composer/standard_honk_composer.test.cpp

```diff
@@ -1,19 +1,18 @@
-#include "standard_honk_composer.hpp"
```

Comment:
The tests that were refactors have enormous diffs. I tried hard to keep the work to a totally formulaic refactor.